src/jbloader/loader/enable_ssh.c

```diff
@@ -21,14 +21,14 @@ void *enable_ssh(void *__unused _)
   char* dropbear_plist = NULL;
   switch (dyld_platform) {
     case PLATFORM_IOS:
-      dropbear_plist = "/cores/binpack/Library/LaunchDaemons/dropbear.plist";
+      dropbear_plist = "/cores/binpack/Library/LaunchDaemons/DropBear/dropbear.plist";
```

Comment:
this change can only be part of the previous commit. otherwise it's completely broken